Gems/PhysX/Code/Tests/DebugDrawTests.cpp

```diff
@@ -96,21 +90,15 @@ namespace PhysXEditorTests
         worldTM.SetRotation(AZ::Quaternion::CreateRotationZ(AZ::DegToRad(45.0f)));
         AZ::TransformBus::Event(boxId, &AZ::TransformBus::Events::SetWorldTM, worldTM);
 
-        UnitTest::TestDebugDisplayRequests testDebugDisplayRequests;
-        AzFramework::EntityDebugDisplayEventBus::Event(boxId, &AzFramework::EntityDebugDisplayEvents::DisplayEntityViewport,
-            AzFramework::ViewportInfo{ 0 }, testDebugDisplayRequests);
-        const AZ::Aabb debugDrawAabbNoNonUniformScale = testDebugDisplayRequests.GetAabb();
+        const AZ::Aabb debugDrawAabbNoNonUniformScale = GetDebugDrawAabb(boxId);
```

Comment:
Thanks for making these updates! Way cleaner now with this useful little helper 🙂